### DISCUSSION TITLE
Why does windows app have scribbled looking icons but online does not?

### DISCUSSION THREAD

**Discussion by SatComEng** [jgraph/drawio-desktop#1401] [General]
I have noticed that at least my Windows installed draw.io app seems to have scribbled or cartoonish looking (non-straight edges) icons where-as the online draw.io app's icons look just fine (straight edges).   Do I have something set wrong and can change this?  
![example](https://github.com/jgraph/drawio-desktop/assets/26719725/832132c7-d360-424e-abf8-5553115322e7)

**Comment by davidjgraph** [comment]
You have sketch mode ticked in desktop

**Reply by SatComEng** [reply]
Ah... Didn't even realize that. Thank you! That fixed it.  Must be the default, as I never turned that on.
